docs/grid_exchange_formats/cgmes/import.md

```diff
@@ -539,6 +541,62 @@ The PowSyBl `LccConverterStation` `PowerFactor` is calculated from the CGMES SSH
 
+
+(cgmes-detailed-dc-model-import)=
+### Detailed DC model
+
+In order to import CGMES DC objects into the IIDM detailed DC model, it is required to set the optional parameter
+`iidm.import.cgmes.use-detailed-dc-model` to `true`.
+
+The following CGMES classes are read and imported: `DCNode`, `DCTopologicalNode`, `DCLineSegment`, `DCSwitch`, `DCBreaker`, `DCDisconnector`,
+`DCGround`, `CsConverter`, `VsConverter`.
+
+#### DCNode, DCTopologicalNode
+
+If the CGMES model is a node/breaker one, then CGMES `DCNode` are imported into PowSyBl [`DC Node`](../../grid_model/network_subnetwork.md#dc-node),
+and CGMES `DCTopologicalNode` are discarded. If the CGMES model is a bus/branch one, then CGMES `DCTopologicalNode` are imported into PowSyBl `DcNode`.
+Attribute mapping is:
+- `NominalV` is copied from EQ `ratedUdc` of a CGMES `ACDCConverter` which is located in the same `DCConverterUnit` as the `DCNode`.
+
+#### DCLineSegment
+
+CGMES `DCLineSegment` are imported into PowSyBl [`DC Line`](../../grid_model/network_subnetwork.md#dc-line), with attribute:
+- `R` is copied from EQ `resistance`.
+
+#### DCSwitch, DCBreaker, DCDisconnector
+
+All CGMES `DCSwitch` are imported into PowSyBl [`DC Switch`](../../grid_model/network_subnetwork.md#dc-switch), with attributes described as follow:
+- `Kind` is defined depending on the CGMES class: it is `BREAKER` for CGMES `DCBreaker`, and it is `DISCONNECTOR` for CGMES `DCSwitch` and `DCDisconnector`.
+- `Open` is defined depending on the associated CGMES `DCTerminal`: if both have SSH `connected` set to `true`, then it is `false`, otherwise it is `true`.
+
+#### DCGround
+
+CGMES `DCGround` are imported into PowSyBl [`DC Ground`](../../grid_model/network_subnetwork.md#dc-ground), with attribute:
+- `R` is copied from EQ `r`.
+
+#### ACDCConverter (CsConverter, VsConverter)
+
+CGMES `CsConverter` are imported into PowSyBl [`Line Commutated Converter`](../../grid_model/network_subnetwork.md#line-commutated-converter),
+and `VsConverter` into [`Voltage Source Converter`](../../grid_model/network_subnetwork.md#voltage-source-converter).
+Common [`AC/DC Converter`](../../grid_model/network_subnetwork.md#acdc-converter) attributes are mapped as follows:
+- `IdleLoss` is copied from EQ `idleLoss`.
+- `SwitchingLiss` is copied from EQ `switchingLoss`.
+- `ResistiveLoss` is copied from EQ `resistiveLoss`.
+- `PccTerminal` is copied from EQ `PccTerminal`.
+- `ControlMode` is `P_PCC` if SSH `pPccControl` is `activePower` (CSC) or `pPcc` (VSC), else it is `V_DC` if `pPccControl` is `dcVoltage` (CSC) or `udc` (VSC).
+- `TargetP` is copied from SSH `targetPpcc`.
+- `TargetVdc` is copied from SSH `targetUdc`.
+
+Specific `CsConverter` attributes are mapped as follows:
+- `ReactiveModel` is always set to `FIXED_POWER_FACTOR`.
+- `PowerFactor` is calculated from SSH `p` and `q` as $\arctan({\frac{p}{\sqrt{p^2 + q^2}}})$. 
+If p and q are equal to 0, the power factor is calculated with $Q = 0.5P$, which gives the value 0.89443.
```

Comment:
There's a typo here: this is not the `arctan` but the `abs` that is used in the code ([here](https://github.com/powsybl/powsybl-core/blob/41824af483f522b8b586b453d28dbb079d235e6d/cgmes/cgmes-conversion/src/main/java/com/powsybl/cgmes/conversion/elements/dc/AcDcConverterConversion.java#L262)).